### PR TITLE
Awesome new conversation list

### PR DESCRIPTION
**Pull request by JPonte**
The old adapter and list view are still being used but should in the near future be also replaced.

#### APK
[Download build #8818](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8818/artifact/build/artifact/wire-dev-PR819-8818.apk)
[Download build #8820](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8820/artifact/build/artifact/wire-dev-PR819-8820.apk)
[Download build #8822](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8822/artifact/build/artifact/wire-dev-PR819-8822.apk)
[Download build #8823](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8823/artifact/build/artifact/wire-dev-PR819-8823.apk)
[Download build #8824](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8824/artifact/build/artifact/wire-dev-PR819-8824.apk)

### CODE REVIEW

**Comment by someoneEsle** [comment]
<Drama mode ON>
Please, _I beg you_, make the font of the subtitles in the conversation list bigger. Much bigger. :sob: They are honestly too small to read without squinting. Also, even after going to my phone's settings to increase the dimension of the overall fonts, the subtitles do not seem to have gotten any bigger. (By the way when you increase the font size, the height of a conversationListRow should also increase or the conversation name and subtitle will partly overlap)

**Comment by JPonte** [comment]
build again

File: app/src/main/scala/com/waz/zclient/views/conversationlist/NewConversationListRow.scala [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+  val avatarInfo = for {
+    z <- zms
+    self <- selfId
+    conv <- conversation
+    memberIds <- z.membersStorage.activeMembers(conv.id)
+    memberSeq <- Signal.future(z.usersStorage.getAll(memberIds)).map(_.flatten)
+  } yield {
+    val opacity =
+      if (memberIds.isEmpty || conv.convType == ConversationType.WaitForConnection || !conv.activeMember)
+        0.25f
```
Comment (hakonbo) [inline app/src/main/scala/com/waz/zclient/views/conversationlist/NewConversationListRow.scala]: Do we have these values in xml somewhere perhaps? If you, I think we should add them.

File: wire-core/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -140,13 +140,13 @@
...
+    <dimen name="wire__text_size__small">12sp</dimen>
     <dimen name="wire__text_size__regular">16sp</dimen>
-    <dimen name="wire__text_size__medium">24dp</dimen>
-    <dimen name="wire__text_size__big">28dp</dimen>
-    <dimen name="wire__text_size__huge">32dp</dimen>
-    <dimen name="wire__text_size__insane">160dp</dimen>
+    <dimen name="wire__text_size__medium">24sp</dimen>
+    <dimen name="wire__text_size__big">28sp</dimen>
+    <dimen name="wire__text_size__huge">32sp</dimen>
+    <dimen name="wire__text_size__insane">160sp</dimen>
```
Comment (hakonbo) [inline wire-core/src/main/res/values/dimens.xml]: Have you checked that switch from `dp` to `sp` works when user has large text activated? Especially the insane one...

File: wire-core/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -896,7 +898,7 @@
     <string name="location_sharing__missing_play_services">Google Play Services is needed for location sharing</string>
 
     <!-- From SyncEngine -->
-    <string name="zms_empty_conversation_name">Empty conversation</string>
+    <string name="zms_empty_conversation_name">Group conversation</string>
```
Comment (hakonbo) [inline wire-core/src/main/res/values/strings.xml]: Don't believe this string is used by the UI

File: wire-core/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1129,4 +1131,41 @@
...
+        <item quantity="other">%1$s missed calls</item>
+    </plurals>
+    <plurals name="conversation_list__pings_count">
+        <item quantity="one">%1$s ping</item>
+        <item quantity="other">%1$s pings</item>
+    </plurals>
+    <plurals name="conversation_list__new_likes_count">
+        <item quantity="other">%1$s ♥</item>
+    </plurals>
+    <string name="conversation_list__unsent_message_long">⚠ Unsent message️</string>
```
Comment (hakonbo) [inline wire-core/src/main/res/values/strings.xml]: Some unintentional character at the end here?

**Comment by red-avtovo** [comment]
Mute badge has different width

STR
A mutes conversation with B
B makes a call to A
A rejects the call
A sees JOIN badge
B hags up
A sees wide badge

506dd59a-5f69-46c1-882b-5e419a6072ed

File: wire-core/src/main/res/values/dimens.xml (hunk 2 above)
Comment (JPonte) [inline wire-core/src/main/res/values/dimens.xml]: I didn't find any issues from what I tested... Maybe we could ask QA to take a look as well on that scenario.

**Comment by JPonte** [comment]
build again

**Comment by JPonte** [comment]
build again

**Pull request merged**